INSTALL.md

```diff
@@ -0,0 +1,163 @@
+# Use Helm to quickly deploy Dragonfly+Nydus on K8S
+# Introduction
+In the cloud-native scenario, the operation of business systems is inseparable from the creation, distribution, and operation of images. In the process of using the current image, the full image needs to be pulled from the warehouse to the local, and then decompressed and mounted by the container engine before it can be provided to the container. In the actual production process, the distribution and loading of a large number of images is often the bottleneck that limits the speed of container startup.
+
+In order to solve the above problems, we introduced the image acceleration solution of Nydus+Dragonfly. However, in the past deployment process, users need to manually install many components, which is costly. To this end, this article will introduce how to use the Kubernetes package management tool Helm to deploy the Dragonfly+Nydus solution on the K8S cluster simply and quickly.
```

Comment:
Maybe give the nydus website link like `[Nydus](https://nydus.dev/)`.

Reply:
Looks cool.